src/formatting.ts

```diff
@@ -1,5 +1,7 @@
-import FormData from "form-data";
 import { Readable } from "stream";
+import { FormData } from "./types/formData";
+
+const FormData = require("form-data");
```

Comment:
Could you add a comment explaining why we're using require? Also might be better to use `// tslint:disable-next-line: no-var-requires` instead of disabling everywhere, as we do in `WebhookEventVerifier`